src/test/regress/expected/isolation_rebalancer_deferred_drop.out

```diff
@@ -34,20 +33,16 @@ step s1-begin:
     BEGIN;
 
 step s1-move-placement:
-        SET citus.defer_drop_after_shard_move TO ON;
-     SELECT master_move_shard_placement((SELECT * FROM selected_shard), 'localhost', 57637, 'localhost', 57638);
+    SELECT master_move_shard_placement((SELECT * FROM selected_shard), 'localhost', 57637, 'localhost', 57638);
 
 master_move_shard_placement
 
 
 step s2-drop-marked-shards:
     SET client_min_messages to DEBUG1;
     SELECT public.master_defer_delete_shards();
-
-master_defer_delete_shards
-
-0
-step s1-drop-marked-shards:
+ <waiting ...>
+step s1-drop-marked-shards: 
```

Comment:
This seems like a change that should not have happened. Is this happening consistently? Given we set `citus.defer_shard_delete_interval=-1` in the default test environment, I don't think it would be caused by the background worker.